pages/api/showEndGameImage.tsx

```diff
@@ -0,0 +1,9 @@
+import Firework from "../../components/math/longestStreak/Firework";
+
+export function showEndGameImage() {
+  let optionOne = <Firework />;
```

Comment:
This should be a ts file not tsx

Reply:
I could only get it to work in a tsx file due to the fact I'm conditionally rendering a component. Is there a better way?